### PR TITLE
Make UI tests pass against 203

### PR DESCRIPTION
**Pull request by abrooksv**
* Also make a few changes aimed to make them more stable

203:
```
SUCCESS: Executed 3 tests in 7m 23s


Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.5/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 7m 25s
3 actionable tasks: 1 executed, 2 up-to-date
```

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt [hunk 1]
```diff
@@ -18,7 +18,7 @@ import java.awt.event.KeyEvent
 import java.time.Duration
 
 fun RemoteRobot.idea(function: IdeaFrame.() -> Unit) {
-    val frame = find<IdeaFrame>()
+    val frame = find<IdeaFrame>(timeout = Duration.ofSeconds(10))
     // FIX_WHEN_MIN_IS_203 remove this and set the system property "ide.show.tips.on.startup.default.value"
```
Comment (hunterwerlla) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt:22]: What happens if we set the property now for 203 and only close if < 203?

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt [hunk 2]
```diff
@@ -25,31 +25,53 @@ fun RemoteRobot.welcomeFrame(function: WelcomeFrame.() -> Unit) {
...
+        } else {
+            selectTab("Customize")
+            findAndClick("//div[@accessiblename='All settings…']")
         }
+
         log.info("Successfully opened the preferences dialog")
     }
 
+    fun selectTab(tabName: String) {
+        if (remoteRobot.ideMajorVersion() <= 202) return
```
Comment (hunterwerlla) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt:64]: nit: for all these checks `FIX_WHEN_MIN_IS_203 remove the if`?

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt (hunk 1 above)
Comment (abrooksv) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt:22]: Eh, I'd rather keep it till we can kill it off 100%

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt (hunk 2 above)
Comment (abrooksv) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/WelcomeFrame.kt:64]: Sure, if this keeps getting bad, may make IDE version impls of an interface ...

**Review by hunterwerlla — APPROVED** [review]

**Review by hunterwerlla — APPROVED** [review]

**Review by hunterwerlla — APPROVED** [review]

**Pull request merged**